Readability.js

```diff
@@ -721,11 +721,11 @@ Readability.prototype = {
           }
         }
 
-        // Remove empty DIV, SECTION, and HEADER nodes
+        // Remove DIV, SECTION, and HEADER nodes without any content(e.g. text, image, video, or iframe).
```

Comment:
```
 If we detect containers with only an hr/br in them, can we reparent the hr/br to the parent of the container and then remove the container, or does that still not help?
```
No, it doesn't help. The [logic] only works if the top candidate is the only child. I think we should take the solution since we could not come up other better ideas.

```
are the hr/br styled to no longer affect the visual appearance of the page?
```
You can see the `<hr>` node is represented as the three dots on the page in the below screenshot. I would say removing the `<hr>` node here is acceptable.

If it is acceptable, let's merge the patch and move on the next issue. :)

<img width="1434" alt="screen shot 2017-02-21 at 4 23 40" src="https://cloud.githubusercontent.com/assets/3013038/23156576/8c1513a4-f852-11e6-8738-859930774157.png">

[logic]: https://github.com/mozilla/readability/blob/master/Readability.js#L925-L931